CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Changed
 - slightly wider account sidebar (so traffic lights look more centered on macOS) #3698
+- remove chatstore global for prevention of possible race condition #3700
```

Comment:
that window variable was just unused see commit message, the store itself is still global, so a changelog entry is not needed yet as this is just a mini refactoring of code.

Reply:
solved by 42be0ebab022193eeadf853653ac14c67b48eaa7